src/components/flash/flash.js

```diff
@@ -305,19 +318,19 @@ class Flash extends React.Component {
           key={ title }
           title={ title }
           open={ this.state.dialogs[title] || false }
-          onCancel={ this.toggleDialog.bind(this, title) }
+          onCancel={ this.toggleDialog(title) }
         >
           { desc }
         </Alert>
       );
 
       // create text for item
-      text = (
+      return (
```

Comment:
`text` has been converted into a `return` statement here, but `return text` is still below.

(also, it looks like the bracket configuration we prefer in an earlier comment is passing linting here